packages/headless/src/features/recommendation/recommendation-actions.ts

```diff
@@ -94,6 +94,8 @@ export const buildRecommendationRequest = (
   organizationId: s.configuration.organizationId,
   url: s.configuration.search.apiBaseUrl,
   recommendation: s.recommendation.id,
+  tab: s.configuration.analytics.originLevel2,
+  referrer: s.configuration.analytics.originLevel3,
```

Comment:
Just do a double check that it's `referrer` and not `referer` (like the HTTP spec). I can never remember which one we use...

Reply:
The [swagger](https://platform.cloud.coveo.com/docs?urls.primaryName=Search%20API#/Search%20V2/searchUsingPost) and [search-ui ](https://static.cloud.coveo.com/searchui/v2.10081/index.html#q=wow&t=All&sort=relevancy) send `referrer` 🤞